README.md

```diff
@@ -48,13 +48,13 @@ A basic example of a custom `defaultDocumentNode` function, to only show the Ifr
 // ./src/defaultDocumentNode.ts
 
 import {type DefaultDocumentNodeResolver} from 'sanity/structure'
-import {Iframe} from 'sanity-plugin-iframe-pane'
+import {Iframe, UrlResolver} from 'sanity-plugin-iframe-pane'
 import {type SanityDocument} from 'sanity'
 
-// Customise this function to show the correct URL based on the current document
-function getPreviewUrl(doc: SanityDocument) {
+// Customise this function to show the correct URL based on the current document and the current studio perspective
+const getPreviewUrl: UrlResolver = (doc, perspective) => {
   return doc?.slug?.current
-    ? `${window.location.host}/${doc.slug.current}`
+    ? `${window.location.host}/${doc.slug.current}?perspective=${perspective.perspectiveStack}`
```

Comment:
It might be useful to know about `import {urlSearchParamPreviewPerspective} from '@sanity/preview-url-secret/constants'`
which ensures it's compatible with `@sanity/presentation`, as well as `next-sanity`'s `defineLive` and `next-sanity/draft-mode`.

```suggestion
    ? `${window.location.host}/${doc.slug.current}?${urlSearchParamPreviewPerspective}=${perspective.perspectiveStack}`
```